.azure-devops/full-pipeline.yml

```diff
@@ -12,3 +12,21 @@ jobs:
       - template: ./steps/devcerts.yml
       - template: ./steps/edgewebview.yml
       - template: ./steps/test.yml
+
+  - job: Mac
+    pool:
+      name: OPX Mac Dogfood Pool
```

Comment:
Why are the names between the Windows, Mac, and Linux pipelines so different between themselves?

Reply:
Those are the names of the agent pools they get the image from.  The windows case is 1ES and was named by jeff.  The mac case was named by Courtney for our custom pools, and the Linux case is a vanilla azure image.